kubernetes/config/common.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: v1
+data:
+  common.libsonnet: |
+    {
+      blobstore: {
+        contentAddressableStorage: {
+          sharding: {
+            hashInitialization: 11946695773637837490,
+            shards: [
+              {
+                backend: { grpc: { address: 'storage-0.storage.buildbarn:8981' } },
+                weight: 1,
+              },
+              {
+                backend: { grpc: { address: 'storage-1.storage.buildbarn:8981' } },
+                weight: 1,
+              },
+            ],
+          },
+        },
+        actionCache: {
+          sharding: {
+            hashInitialization: 14897363947481274433,
+            shards: [
+              {
+                backend: { grpc: { address: 'storage-0.storage.buildbarn:8981' } },
+                weight: 1,
+              },
+              {
+                backend: { grpc: { address: 'storage-1.storage.buildbarn:8981' } },
+                weight: 1,
+              },
+            ],
+          },
+        },
+      },
+      browserUrl: 'http://localhost:7980',
+      maximumMessageSizeBytes: 16 * 1024 * 1024,
+      httpListenAddress: ':7980'
+    }
+
+kind: ConfigMap
+metadata:
+  name: common
+  namespace: buildbarn
```

Comment:
Misc: your editor seems to remove trailing newlines from these files. Could you please re-add those? Files with missing trailing newlines are annoying to work with. For example, concatenating them would cause the resulting data to be malformed.